### ISSUE TITLE
Examples and docs

### ISSUE THREAD

**Issue by javadba** [jmd1011/parquet-readers#5]
This repo is linked to by several places so kind of hoped for some guidance..

**Comment by TiarkRompf** [comment]
> No Docs, no examples, and uncertain even how to run the unit tests
>
> Self explanatory from the title. I'm assuming this is abandonware.. 

Just because someone puts code out there, it's not like you're entitled to support.

(Edited to reflect the original issue title and comment I replied to)

**Comment by javadba** [comment]
Fair enough - I was not asking for that - but only a self contained project: when I "put one out there" I do include those.  This project has been pointed to / linked to with some regularity so the expectations are well that it's kind of minimally documented. It's not - so fair  indication for  others interested in this project they're completely on their own.

**Comment by TiarkRompf** [comment]
> when I "put one out there" I do include those

A quick look at your GitHub profile (https://github.com/javadba?utf8=✓&tab=repositories&q=&type=source&language=) indicates otherwise.
